packages/compress/zlib/package.mk

```diff
@@ -11,7 +11,3 @@ PKG_DEPENDS_HOST="cmake:host"
 PKG_DEPENDS_TARGET="toolchain"
 PKG_LONGDESC="A general purpose (ZIP) data compression library."
 PKG_TOOLCHAIN="cmake-make"
-
```

Comment:
Please drop changes to packages/compress/zlib/package.mk. These are needed.

Reply:
done